src/avram/database/column_info.cr

```diff
@@ -1,26 +1,28 @@
 module Avram
-  struct Database::ColumnInfo
-    include DB::Serializable
+  class Database
+    struct ColumnInfo
```

Comment:
This avoids a potential issue where this file gets required before the `Database` class does causing this namespace to be treated like a module.